app/views/qsets/_question_page_header_small_screen.html.erb

```diff
@@ -18,3 +18,8 @@
     <%= render "all_mine_other_filter" %>
   </div>
 </div>
+<div class="row center">
+  <div class="col-xs-12" id="top-padding">
```

Comment:
As for this ID name, please see https://github.com/penzance/askup/pull/85/files#r60822374.